scripts/lib.star

```diff
@@ -298,7 +299,7 @@ def publish_storybook_step(edition, ver_mode):
         'name': 'publish-storybook',
         'image': publish_image,
         'depends_on': [
-            'build-storybook',
+            'verify-storybook',
```

Comment:
Is this right? it looks like it's part of the build step, not a seperate drone step

Reply:
@torkelo You're right, that change shouldn't be made. I fixed it.

Reply:
thanks @aknuds1